roles/openshift_docker_gc/tasks/main.yaml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: Ensure that docker-gc will have nodes to run on
+  import_role:
+    name: openshift_master
+    tasks_from: ensure_nodes_matching_selector.yml
+  vars:
+    openshift_master_ensure_nodes_selector: "{{ r_docker_gc_node_selectors }}"
```

Comment:
Probably need the map_to_pairs filter the other examples use.

Reply:
I noticed that a few used that and a few used raw inputs. I'm a little leary to switch to ``map_to_pairs`` right now for this. Do you think it's a must have?

Reply:
I have not investigated it.  Have you tested with this version?

Reply:
Yes. It was used previously. This change really just uses the same options and turns them on by default rather than them being disabled.